src/test/java/com/google/sps/GameUtilTest.java

```diff
@@ -61,9 +61,12 @@ public void createGame_EmptyGameName() {
   @Test
   public void createGame_NullDatastore() {
 
-    Entity actual = GameUtils.createGameEntity("game", null);
-
-    Assert.assertEquals(null, actual);
+    Entity actual = null;
+    try{
+        actual = GameUtils.createGameEntity("game", null);
+    }catch(NullPointerException e){
+        Assert.assertEquals(null, actual);
+    }
```

Comment:
here (and everywhere else in this PR) we'll want to actually assert that the exception was thrown.

What you did is technically a correct way of accomplishing this goal, but it's not intuitive and there is a more elegant way to do this:

https://howtodoinjava.com/junit5/expected-exception-example/

Let me know if you have any questions!

Reply:
Okay, that's done.